app/models/api_ability.rb

```diff
@@ -0,0 +1,13 @@
+class ApiAbility
+  include CanCan::Ability
+
+  def initialize(member)
+    member ||= Member.new
+    if member
+      #
+    else
```

Comment:
aici la fel ca la ability si aici trebuia sa definesti toate alea nu in ability .... retardism si codu asta!